content/en/docs/releases/bugs/index.md

```diff
@@ -65,16 +69,40 @@ containing:
 
 * Current and previous logs from all Istio components and sidecar
 
-* Istiod logs:
+    * Istiod logs:
+
+        {{< text bash >}}
+        $ kubectl logs -n istio-system -l app=istiod
+        {{< /text >}}
+
+    * Ingress Gateway logs:
+
+        {{< text bash >}}
+        $ for ns in $(kubectl get ns -o jsonpath='{.items[*].metadata.name}') ; do kubectl logs -l istio=ingressgateway -n $ns ; done
+        {{< /text >}}
+
+    * Egress Gateway logs:
+
+        {{< text bash >}}
+        $ for ns in $(kubectl get ns -o jsonpath='{.items[*].metadata.name}') ; do kubectl logs -l istio=egressgateway -n $ns ; done
+        {{< /text >}}
+
+    * Sidecar logs:
+
+        {{< text bash >}}
+        $ for ns in $(kubectl get ns -o jsonpath='{.items[*].metadata.name}') ; do kubectl logs -l service.istio.io/canonical-revision -c istio-proxy -n $ns ; done
+        {{< /text >}}
+
+* All Istio configuration artifacts:
 
     {{< text bash >}}
-    $ kubectl logs -n istio-system -l app=istiod
+    $ kubectl get istio-io --all-namespaces -o yaml
     {{< /text >}}
 
-* All Istio configuration artifacts:
+* Output of istioctl analyze:
```

Comment:
Thanks for doing this.
Can we move this to the top of the list.
If the user finds the problem in the analyze output all the rest of the steps are saved.

Reply:
Great point, will do!